lib/cisco_node_utils/interface.rb

```diff
@@ -161,23 +148,52 @@ def default_encapsulation_dot1q
       config_get_default('interface', 'encapsulation_dot1q')
     end
 
+    # encapsulation_profile_vni
+    #   cli: service instance 1 vni
+    #          encapsulation profile vni_500_5000 default
+    #  type: [['<instance>', '<profile>'], ['1', 'vni_500_5000']]
     def encapsulation_profile_vni
-      val = config_get('interface', 'encapsulation_vni', @name)
-      debug "val from get is #{val}"
-      return '' if val.nil?
-      val.first.strip
-    end
-
-    def encapsulation_profile_vni=(val)
-      if val.nil?
-        config_set('interface', 'encapsulation_vni_del', @name, val)
-      else
-        config_set('interface', 'encapsulation_vni_add', @name, val)
+      profiles = []
+      svc_list = config_get('interface', 'all_service_instances', @name)
+      return profiles if svc_list.nil?
+      svc_list.each do |svc|
+        profile = config_get('interface', 'encapsulation_profile_vni',
```

Comment:
`encapsulation_profile_vni` shows up under interface_service.yaml in your diffs...